### ISSUE TITLE
batch size, lr, and schedule.

### ISSUE THREAD

**Issue by KaimingHe** [roytseng-tw/Detectron.pytorch#32]
According to the documentation, if I understand correctly, in some settings you changed the batch size, and thus lr proportionally, but you did not change the schedule (in terms of "iterations"). You should scale the schedule proportionally and let the solver see the same total number of images. To match curves, the x-axis should also be # of images (or equivalently, epochs), but not iterations.

**Comment by roytseng-tw** [comment]
Yes, besides scaling the learning rate, I should also scale the training schedule. And to match loss curves, it's more meaningful to base on # of images have been seen. I'll improve that soon.

**Comment by roytseng-tw** [comment]
Fixed and closing the issue.
